src/scheduler/calculateDelayFromInterval.ts

```diff
@@ -0,0 +1,18 @@
+import { DateTime } from 'luxon';
+import { max, toInteger } from 'lodash';
+import humanInterval from 'human-interval';
+import { IntervalSchedule } from '../job/MomoJob';
+
+export function calculateDelayFromInterval(schedule: IntervalSchedule, jobLastStarted: string | undefined): number {
```

Comment:
I guess this function( or at least the logic) was moved into the ExecutableIntervalSchedule class and is now unused

Reply:
Good catch!